src/Components/CategoryPageProducts/CategoryPageProducts.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import { useCartWishlist } from "../../context/cart-wishlist-context";
+import Heading from "../Heading/Heading";
+import ProductCard from "../ProductCard/ProductCard";
+import "../ProductsPage/ProductsPage.css";
+
+function CategoryPageProducts({ categoryName }) {
+  const { state } = useCartWishlist();
```

Comment:
Just a suggestion,
Here if you want to retrieve just the productsArr, then you could do something like this.

```
const {state: {productsArr}} = useCartWishlist();
```